ci_cd/tasks/update_deps.py

```diff
@@ -2,7 +2,6 @@
 
 Update dependencies in a `pyproject.toml` file.
 """
-# pylint: disable=duplicate-code
```

Comment:
Does this mean that ruff accepts duplicate code?

Reply:
Indeed, it doesn't implement pylint's `duplicate-code` check yet. See [this issue](https://github.com/astral-sh/ruff/issues/970) for all the rules ruff does and doesn't implement from pylint.

In this case, I'm not too worried (especially because it was already ignored), but it's something to keep an eye out for indeed.